# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1473749982 18000
#      Tue Sep 13 01:59:42 2016 -0500
# Node ID a7d6c66e91e4a49c6cbc21dc1c5363df8b71802a
# Parent  6bf1cbb90705070cea9f83821cd16bc2cd7e9c08
Rid dashed termoption warning for tikz terminal of gnuplot_toolkit (bug #49003).

* __gnuplot_drawnow__.m: Only send termoption dashed to gnuplot pipe if gnuplot
  "has_termoption_dashed" and does not have the more recent and preferred
  "dashtype".

diff --git a/scripts/plot/util/__gnuplot_drawnow__.m b/scripts/plot/util/__gnuplot_drawnow__.m
--- a/scripts/plot/util/__gnuplot_drawnow__.m
+++ b/scripts/plot/util/__gnuplot_drawnow__.m
@@ -334,17 +334,18 @@ function enhanced = gnuplot_set_term (pl
     elseif (any (strcmp (term, flickering_terms)))
       fprintf (plot_stream, "%s\n", term_str);
       if (nargin == 5)
         if (! isempty (file))
           fprintf (plot_stream, "set output '%s';\n", file);
         endif
       endif
     endif
-    if (__gnuplot_has_feature__ ("has_termoption_dashed"))
+    if (__gnuplot_has_feature__ ("has_termoption_dashed")
+        && ! __gnuplot_has_feature__ ("dashtype"))
       fprintf (plot_stream, "set termoption dashed\n");
     endif
   else
     ## gnuplot will pick up the GNUTERM environment variable itself
     ## so no need to set the terminal type if not also setting the
     ## figure title, enhanced mode, or position.
   endif
 
